### DISCUSSION TITLE
Installation on Linux

### DISCUSSION THREAD

**Discussion by Aref20** [saleor/saleor#10314] [Deployment]
Where can I find an installation guide for Linux?

**Comment by zimmerx** [comment]
Nowhere if you want to use the latest release :) and it's the most awful documentation which I've ever read. But for the easiest access, you can use already pre-built Docker image. If you want to use it in production then be ready to become DevOps to run it smoothly. last but not least :) Money talks, you can pay for the Cloud version.

**Reply by timuric** [reply]
Hey @zimmerx we are revamping our docs, and this is in our backlog already, any other feedback about docs is welcome!

https://github.com/saleor/saleor-docs/issues
